<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RTLS Controller</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/icons/DWctrler.ico</normaloff>:/icons/DWctrler.ico</iconset>
  </property>
  <widget class="GraphicsWidget" name="graphicsWidget">
   <widget class="QDockWidget" name="channel_dw">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>600</y>
      <width>960</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>98</width>
      <height>60</height>
     </size>
    </property>
    <property name="allowedAreas">
     <set>Qt::NoDockWidgetArea</set>
    </property>
    <widget class="ChannelSettingsWidget" name="channel_settings"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="AnchorMenu" name="anchorMenu">
    <property name="title">
     <string>&amp;Advanced</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;RTLS</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
   </widget>
   <addaction name="menuConfig"/>
   <addaction name="anchorMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="anchorList_dw">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>44</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Anchors</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="AnchorListWidget" name="anchor_list"/>
  </widget>
  <widget class="QDockWidget" name="anchorProps_dw">
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>550</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="windowTitle">
    <string>Anchor Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="AnchorPropertiesWidget" name="anchor_properties">
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDockWidget" name="inspector_dw">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable</set>
   </property>
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="ModelInspectorWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="viewSettings_dw">
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>View Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="ViewSettingsWidget" name="view_settings"/>
  </widget>
  <widget class="QDockWidget" name="minimap_dw">
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Minimap</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="MinimapView" name="minimap"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>Channel Config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ViewSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ViewSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphicsWidget</class>
   <extends>QWidget</extends>
   <header>GraphicsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MinimapView</class>
   <extends>QWidget</extends>
   <header>MinimapView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnchorPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>anchorpropertieswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnchorListWidget</class>
   <extends>QWidget</extends>
   <header>anchorlistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModelInspectorWidget</class>
   <extends>QWidget</extends>
   <header>ModelInspectorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnchorMenu</class>
   <extends>QMenu</extends>
   <header>AnchorMenu.h</header>
  </customwidget>
  <customwidget>
   <class>ChannelSettingsWidget</class>
   <extends>QWidget</extends>
   <header>channelsettingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
